components/plugins/PluginsLists.vue

```diff
@@ -0,0 +1,394 @@
+                v-for="category in categories"
+                :key="category"
+                :class="{ 'active': category === activeCategory }"
+                @click="setActiveCategory(category)"
+                class="m-1 rounded-button"
+            >
+                {{ capitalize(category) }}
+            </button>
+        </div>
+        <div class="row my-4" data-aos="fade-right">
+            <div class="col-12 search-input position-relative" v-click-outside="handleClickOutside">
+                <input type="text" class="form-control form-control-lg" placeholder="Search across +400 of plugins" v-model="searchQuery" @focus="handleFocus">
+                <Magnify class="search-icon" />
+                <div class="search-list" v-if="searchMenu">
+                    <div v-if="searchResults.length" v-for="result in searchResults">
+                        <NuxtLink :href="`/plugins/${result.name}`">
+                            <div class="slug">
+                                <span :class="{first: index === 0}"  v-for="(item, index) in breadcrumb(result.group)" :key="item" >{{ item }}</span>
+                            </div>
+                            <div class="result rounded-3">
+                                <div>
+                                    <h5 v-html="result.title"></h5>
+                                    <p v-html="result.description" class="search-result-extract"/>
+                                </div>
+                                <ArrowRight />
+                            </div>
+                        </NuxtLink>
+                    </div>
+                    <div v-if="searchResults && !searchResults.length" class="alert alert-warning mb-0" role="alert">
+                        No results found for the current search
+                    </div>
+                </div>
+            </div>
+            <div class="col-lg-4 col-md-6 mb-3" v-for="plugin in plugins" :key="plugin.id">
+                <PluginsPluginCard :plugin="plugin" />
+            </div>
+            <div class="d-flex justify-content-between pagination-container">
+                <div class="items-per-page">
+                    <select class="form-select" aria-label="Default select example" v-model="itemsPerPage">
+                        <option :value="15">15</option>
+                        <option :value="25">25</option>
+                        <option :value="50">50</option>
+                    </select>
+                </div>
+                <div class="d-flex justify-content-between align-items-center">
+                    <CommonPagination
+                        :totalPages="totalPages"
+                        @on-page-change="changePage"
+                        v-if="totalPages > 1"
+                    />
+                    <div class="d-flex align-items-baseline" v-if="totalPlugins > itemsPerPage">
+                        <span class="total-pages">Total {{ totalPlugins }}</span>
```

Comment:
How about `{{totalPlugins}} Plugins`?

Reply:
![image](https://github.com/kestra-io/docs/assets/37618489/cfa321bd-328f-4e6e-a4da-df994701e002)
I think I will add ':' to align with what format we use on the UI